[2021-06-25 15:14:51,403] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available 2021-06-24T00:00:00+00:00 [queued]>
[2021-06-25 15:14:51,410] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available 2021-06-24T00:00:00+00:00 [queued]>
[2021-06-25 15:14:51,410] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-25 15:14:51,410] {taskinstance.py:1068} INFO - Starting attempt 3 of 3
[2021-06-25 15:14:51,410] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-25 15:14:51,413] {taskinstance.py:1087} INFO - Executing <Task(HttpSensor): is_api_available> on 2021-06-24T00:00:00+00:00
[2021-06-25 15:14:51,415] {standard_task_runner.py:52} INFO - Started process 31693 to run task
[2021-06-25 15:14:51,418] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'user_processing', 'is_api_available', '2021-06-24T00:00:00+00:00', '--job-id', '8', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/var/folders/9t/2jnn4bv95w9552ldckktgq900000gn/T/tmp6lbt9n7t', '--error-file', '/var/folders/9t/2jnn4bv95w9552ldckktgq900000gn/T/tmpyr37d34x']
[2021-06-25 15:14:51,418] {standard_task_runner.py:77} INFO - Job 8: Subtask is_api_available
[2021-06-25 15:14:51,431] {logging_mixin.py:104} INFO - Running <TaskInstance: user_processing.is_api_available 2021-06-24T00:00:00+00:00 [running]> on host saidamirs-macbook-pro.local
[2021-06-25 15:14:51,443] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2021-06-24T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-06-24T00:00:00+00:00
[2021-06-25 15:14:51,444] {http.py:101} INFO - Poking: api/
[2021-06-25 15:14:51,445] {base.py:69} INFO - Using connection to: id: user_api. Host: https://randomuser.me/, Port: None, Schema: , Login: , Password: None, extra: {}
[2021-06-25 15:14:51,446] {http.py:140} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2021-06-25 15:14:51,516] {local_task_job.py:151} INFO - Task exited with return code Negsignal.SIGABRT
